panel/src/components/Forms/Field/BuilderField.vue

```diff
@@ -165,6 +170,21 @@ export default {
     };
   },
   computed: {
+    draggableOptions() {
+      return {
+        id: Math.random(),
```

Comment:
@afbora I copied this from your draft. I am wondering now whether we could find a cleaner solution for this than a random number.

Reply:
Absolutely, I just needed a unique id. I was surprised that it even got this far 😅

Reply:
Could we just use the field's `name`? Cause a field name should be unique for a page (view), right?

Reply:
Actually it was the field name that came to my mind, but if there is another builder with the same name in the nested area, I thought it would conflict.

Reply:
Always those nested fields...

Reply:
Maybe silly idea, but we could use `endpoints.field` - that should be unique even for nested fields.

Reply:
I went for `this._uid`